elements/rh-table/rh-table.ts

```diff
@@ -138,6 +143,27 @@ export class RhTable extends LitElement {
     if (this.#table && this.#summary) {
       this.#table.setAttribute('aria-describedby', 'summary');
     }
+
+    /* Auto-assign `data-label` attributes based on column headers */
+    if (this.#table?.tHead && this.#colHeaders && this.#rows) {
+      for (const row of this.#rows) {
+        if (!row) {
+          continue;
+        }
+
+        const cells = row.querySelectorAll(':is(td, th)');
+        let headerIndex = 0;
```

Comment:
instead of tracking and incrementing this, I suggest using the `forEach` method on the NodeList, and including the second `index` parameter in the callback

Reply:
Ah ha! I had it this way originally 😉

I will see about looping w/ `forEach`

Reply:
@bennypowers can you give this a re-review?

Reply:
Pushed a few commits dealing with this function